quickwit-proto/src/cluster.rs

```diff
@@ -4,284 +4,36 @@
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Member {
     //// Member ID.　A string of the UUID.
-    #[prost(string, tag = "1")]
+    #[prost(string, tag="1")]
```

Comment:
I don't think you wanted to take into account these changes in this PR.

Reply:
No, but these are from the generated code, there should be no difference